README.md

```diff
@@ -32,69 +36,139 @@ Create `exams.sql` file with `INSERT INTO` query that adds entries to `exams` ta
 Write a query that outputs `year`, `modifier`, `student_cnt` columns that specify the number of students (`student_cnt`) in a class for all classes.
 
 ```sql
-PASTE YOUR CODE HERE
+SELECT year, modifier, student_cnt FROM (SELECT class_id, count(*) AS student_cnt  FROM groups GROUP BY class_id)
+INNER JOIN class
+on class_id == class.id
 ```
 
 ### 2. Find the number of students for every year
 
 Write a query that outputs `year`, `student_cnt` columns that specify the number of students (`student_cnt`) in a class for each year (`year`).
 
 ```sql
-PASTE YOUR CODE HERE
+SELECT year, count(*) AS student_cnt FROM(
+SELECT * FROM groups
+INNER JOIN class
+on class_id == class.id
+) GROUP BY year
 ```
 
 ### 3. Find unique subjects that students learn for a given year
 
 Write a query that outputs `name`, `surname`, `unique_lesson_cnt`, where `unique_lesson_cnt` is the number of **unique** lessons that a student identified by `name`, `surname` has.
 
 ```sql
-PASTE YOUR CODE HERE
+SELECT name, surname, count(*) as unique_lesson_cnt FROM students
+JOIN groups ON groups.student_id == students.id
+JOIN class ON groups.class_id == class.id
+JOIN (
+SELECT class_id as clid FROM timetable
+GROUP BY  class_id, lesson_id
+) ON clid == class.id
+GROUP BY students.id 
 ```
 
 ### 4. Find how many distinct lessons have exams
 
 Write a query that outputs `name` and `lesson_cnt`, where `lesson_cnt` is the number of **unique** lessons (`name`) that have exams.
 
 ```sql
-PASTE YOUR CODE HERE
+SELECT name FROM exams
+JOIN lessons ON lessons.id == exams.lesson_id
+GROUP BY lesson_id
 ```
 
 ### 5.1. Find an average grade for each exam
 
 Write a query that outputs `year`, `modifier`, `lesson` `average_grade`, where `average_grade` is an average grade for each class (identified by `year` and `modifier`) and a subject (identified by `lesson`).
 
 ```sql
-PASTE YOUR CODE HERE
+SELECT year, modifier, lessons.name, AVG(exams.grade) FROM timetable
+JOIN class ON timetable.class_id == class.id
+JOIN lessons ON lessons.id == timetable.lesson_id
+JOIN groups ON groups.class_id == class.id
+JOIN students ON students.id == groups.student_id
+JOIN exams ON students.id == exams.student_id
+GROUP BY year, modifier, lessons.name
 ```
 
 ### 5.2 Find the number of students that passed/failed exams
 
 Write a query that finds the number of students that passed (4+ grade) and failed (<4 grade) the exams.
 
 ```sql
-PASTE YOUR CODE HERE
+SELECT * FROM (
+SELECT count(*) as PASSES FROM exams WHERE grade >=4
+)
+JOIN (SELECT count(*) as FAILED FROM exams WHERE grade <4)
+ON 0 == 0
```

Comment:
Instead of `ON 0 == 0` you can write `CROSS JOIN` instead of `JOIN`. Read about cross join in the documentation.